src/Input/index.scss

```diff
@@ -28,6 +28,10 @@
     display: flex;
     align-items: center;
 
+    // long, unbroken lines of text should horizontal scrol if they exceed
+    // the available width
+    overflow-x: scroll;
```

Comment:
Is there any way that we can make this wrap? I think scrolling still makes it difficult to edit something like certificates.

Reply:
That's a good point - we could `break-word` instead. This won't respect line breaks pasted in, but it looks like in `banking` we don't respect line breaks anyway and render the `\n` escapes in the field value